lib/tasks/tenantize_task.rake

```diff
@@ -1,6 +1,5 @@
 # frozen_string_literal: true
 
-# rubocop:disable Metrics/LineLength
```

Comment:
if you delete this line, you also need to remove the `rubocop:enable Metrics/LineLength` on line 20